LEGO1/mxpresenter.cpp

```diff
@@ -0,0 +1,134 @@
+#include "mxpresenter.h"
+
+// OFFSET: LEGO1 0x1000bee0 STUB
+void MxPresenter::DoneTickle()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x100b4d50
+void MxPresenter::Init()
+{
+  m_unk0x8 = 0;
+  m_action = NULL;
+  m_unk0x18 = 0;
+  m_unk0x3c = 0;
+  m_unk0xc = 0;
+  m_unk0x10 = 0;
+  m_unk0x14 = 0;
+}
+
+// OFFSET: LEGO1 0x100b4fc0 STUB
+void MxPresenter::ParseExtra()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c070 STUB
+MxPresenter::~MxPresenter()
+{
+  // TODO
+}
```

Comment:
This is the first element of the vtable.
I don't think `MxPresenter` has overridden its destructor.

Reply:
I think I agree. 0x1000C070 is the scalar deleting destructor. Judging by the generated code, the destructor itself is empty (apart from the obvious call to destruct the aggregated MxCriticalSection). Whether user-defined in header or just compiler generated is a coin-toss.